packages/internals/src/engine-commands/queryEngineCommons.ts

```diff
@@ -75,11 +76,24 @@ export function loadNodeAPILibrary(queryEnginePath: string) {
   return pipe(
     E.tryCatch(
       () => load<NodeAPILibraryTypes.Library>(queryEnginePath),
-      (e) => ({
-        type: 'connection-error' as const,
-        reason: 'Unable to establish a connection to query-engine-node-api library',
-        error: e as Error,
-      }),
+      (e) => {
+        const error = e as Error
+        const defaultErrorMessage = `Unable to establish a connection to query-engine-node-api library.`
+        const proposedErrorFixMessage = match(error.message)
+          .when(
+            (errMessage) => errMessage.includes('openssl'),
+            () => {
+              return ` Have you tried installing 'openssl'?`
```

Comment:
Seems that the previous comments still apply here
https://github.com/prisma/prisma/pull/13736#discussion_r901492117

Reply:
This is the last blocking part before merging

Reply:
It now says `It seems there is a problem with your OpenSSL installation!`